src/general_harvester.py

```diff
@@ -51,7 +52,7 @@ def __init__(
         self.web3 = web3
         self.keeper_key = keeper_key
         self.keeper_address = keeper_address
-        self.keeper_acl = self.web3.eth.contract(
+        self.keeper_acl: Contract = self.web3.eth.contract(
```

Comment:
why are we typing this member?

Reply:
For better type inference :)

Reply:
But not typing the base_usd_oracle? Seemed inconsistent

Reply:
Ok, I can do that. Was just working with keeper_acl, so decided to type it properly

Reply:
Done